hooks/lists/clubs.ts

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from '@tanstack/react-query'
+import { clubs } from 'dummy-api/register/lists'
+import { returnAfterTime } from 'utils/return-after-time'
+
+export const useClubsList = (searchValue: string) =>
+  useQuery({
+    queryKey: ['club-list', searchValue],
+    queryFn: returnAfterTime(clubs, 200),
+    enabled: !!searchValue,
+    refetchOnWindowFocus: false,
+    initialData: [{ id: '', name: '', gender: 'MALE', teams: [] }],
+  })
```

Comment:
enabled shouldn't depend on search value because maybe somewhere later we'll use a whole clubs lists

I'll correct this later with other swagger types so this uses a filters object infered from query params

```suggestion
export const useClubsList = (searchValue?: string, enabled?: boolean) =>
  useQuery({
    queryKey: ['club-list', searchValue],
    queryFn: returnAfterTime(clubs, 200),
    enabled,
    refetchOnWindowFocus: false,
    initialData: [{ id: '', name: '', gender: 'MALE', teams: [] }],
  })
```